src/main/webapp/app/shared/sidebar/sidebar.component.html

```diff
@@ -23,6 +23,9 @@
                             [routeParams]="routeParams"
                             [courseId]="courseId"
                             [itemSelected]="itemSelected"
+                            [showAddOptions]="sidebarData.sidebarType === 'conversation'"
+                            [showLeadingIcon]="sidebarData.sidebarType === 'conversation'"
```

Comment:
please add an extra key for `showLeadingIcon` default value should be false; and true for conversation
-->  `[showAddOptions]="sidebarData. showLeadingIcon"`